LayoutTests/imported/w3c/web-platform-tests/html-aam/roles-contextual.html

```diff
@@ -34,6 +34,9 @@
   <footer data-testname="el-footer" aria-label="x" class="ex-generic">x</aside>
 </nav>
 <footer data-testname="el-footer-ancestorbody" data-expectedrole="contentinfo" class="ex">x</footer>
+<main>
+  <footer data-testname="el-footer-ancestormain" data-expectedrole="generic" class="ex">x</header>
```

Comment:
Do we need `class="ex-generic"` here instead?

https://github.com/web-platform-tests/wpt/pull/44080#discussion_r1458126371